yarn.lock

```diff
@@ -8272,6 +8249,21 @@ execa@^7.0.0:
     signal-exit "^3.0.7"
     strip-final-newline "^3.0.0"
 
+execa@^8.0.1:
```

Comment:
Have you considered going with `^7.0.0` and aligning with an already existing module to avoid this extra package? 🤔

Reply:
renovatebot would update it right away 🤷

Reply:
Fair point, if we don't add an explicit rule. 🤔 
And what about the evil `resolutions`? 🤔 🙈

Reply:
> And what about the evil `resolutions`? 🤔 🙈

Will likely break things for some of the dependencies as `execa` switched to ESM-only in v6 and we have some dependencies use v5. The only thing we gain would be 80kb on the install size. If install size was that critical that we need to save 80kb I'm sure we can find many more impactful candidates. Besides it'll only take one transitive dependency to update to v8 to start sharing this version.

Reply:
Gotcha, I didn't check that they switched to ESM on v6. 🙈 
Forget this message then. 👌